docs/events/gsoc/2024/gradle-build-server-android.md

```diff
@@ -107,8 +150,11 @@ Throughout the program I have collaborated with all of my mentors and gained inv
 
 My work on this project, significantly expanded my skill set, encompassing JSON-RPC, Language Server Protocol, Build Server Protocol, Gradle API, Gradle Tooling API, testing methodologies and merging techniques with git. I successfully applied these learnings to produce the mentioned outcomes.
 
+If anyone is interested to continue the work and bring further support for Android projects in the [GradleBuildServer](https://github.com/microsoft/build-server-for-gradle) then you can get started [here](https://github.com/microsoft/build-server-for-gradle/labels/android). 
```

Comment:
Please also make sure to reference the key issues, the linked list will be a moving target

Reply:
Will linking [this](https://github.com/microsoft/build-server-for-gradle/pull/173#issuecomment-2308392453) comment work or should I list all the issues individually?